source/extensions/health_checkers/http/health_checker_impl.cc

```diff
@@ -86,9 +88,9 @@ HttpHealthCheckerImpl::HttpHealthCheckerImpl(
       total += bytes.size();
     }
     if (total > response_buffer_size_) {
-      throw EnvoyException(fmt::format(
```

Comment:
Is this change intentional?

Reply:
Yes, trying to avoid direct throw and replace with the macro.

Reply:
We use throw elsewhere in this file. The change looks out of scope for what you need to accomplish. I would just omit it.

Reply:
done